src/notebooks/outputs/tracebackFormatter.unit.test.ts

```diff
@@ -46,7 +46,7 @@ suite(`Notebook trace formatter`, function () {
         const expected = [
             '[0;31m---------------------------------------------------------------------------[0m',
             '[0;31mNameError[0m                                 Traceback (most recent call last)',
-            "[1;32merror.ipynb Cell 1[0m in [0;36m<cell line: 2>[0;34m()[0m\n[1;32m      <a href='vscode-notebook-cell:error.ipynb#W0sZmlsZQ%3D%3D?line=0'>1</a>[0m [39m1[39m\n[0;32m----> <a href='vscode-notebook-cell:error.ipynb#W0sZmlsZQ%3D%3D?line=1'>2</a>[0m x\n",
+            "[1;32merror.ipynb Cell 1[0m line [0;36m<cell line: 2>[0;34m()[0m\n[1;32m      <a href='vscode-notebook-cell:error.ipynb#W0sZmlsZQ%3D%3D?line=0'>1</a>[0m [39m1[39m\n[0;32m----> <a href='vscode-notebook-cell:error.ipynb#W0sZmlsZQ%3D%3D?line=1'>2</a>[0m x\n",
```

Comment:
Should we support both the latest ipykernel format as well as the previous one?

Reply:
this is the result to display

Reply:
To clarify, does our regex now only match on the results from newer ipykernel versions and not results from older versions as a result of this PR?

Reply:
no, it should still match previous versions as well.